integration-tests/opentelemetry/src/main/java/io/quarkus/it/opentelemetry/ExporterResource.java

```diff
@@ -3,29 +3,44 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
 import javax.inject.Inject;
+import javax.inject.Singleton;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
 
 import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter;
 import io.opentelemetry.sdk.trace.data.SpanData;
 
-@Path("/export")
```

Comment:
I'd prefer the explicit export or spans. 
An empty path might be mixed with the resource being tested.

Reply:
The `/export` is at the method level (because we also have a `/reset`). This is consistent with the exporters we have in the other integration-tests modules.

Reply:
ok